include/exadg/incompressible_navier_stokes/spatial_discretization/operator_projection_methods.cpp

```diff
@@ -219,21 +219,6 @@ OperatorProjectionMethods<dim, Number>::setup_solver_pressure_poisson()
   bool constexpr compute_eigenvalues         = false;
   bool const use_preconditioner =
     this->param.preconditioner_pressure_poisson != PreconditionerPressurePoisson::None;
-  std::string name;
```

Comment:
this is an example how we skip now the parsing step enum -> `std::string` required for `dealii::SolverSelector`.